### PR TITLE
Adds proc to update ssl tls keys and cert for OMR

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OCPBUGS-2064

Link to docs preview:
https://65486--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-creating-registry#mirror-registry-ssl-cert-replace_installing-mirroring-creating-registry

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/mirror-registry-ssl-cert-replace.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+----
+$ cp ~/ssl.key $QUAY/quay-config
+----
+
+. Restart the `QUAY` application pod by entering the following command:
++
+[source,terminal]
+----
+$ systemctl restart quay-app 
+----
```
Comment (huweihua-redhat) [inline modules/mirror-registry-ssl-cert-replace.adoc:58]: The steps of certificate update work for me.

But I think there is one thing  should be changed.
  
Screenshot from 2023-10-07 15-16-19

For every command, there is a copy button in the end.  When you click this button,  the whole string in the code block will be copied. That means the `$` will be copied too.  I have to remove `$` when I paste the copied string in my terminal. 
```
[whu@whu]$ $ systemctl restart quay-app
bash: $: command not found...
```
If we much offer this copy button,  we should remove all `$`.

Comment (huweihua-redhat) [reply modules/mirror-registry-ssl-cert-replace.adoc:58]: Oh, I found this copy button is not offered by our code. Maybe customer know what is the correct operation. 

Anyway, the content looks good to me.

File: modules/mirror-registry-ssl-cert-replace.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
+// module included in the following assembly:
+//
+// * installing-mirroring-creating-registry.adoc
```
Comment (bergerhoffer) [inline modules/mirror-registry-ssl-cert-replace.adoc:3]: This should be the full path to the assembly

**Review by bergerhoffer — COMMENTED** [review]
A few things!

File: modules/mirror-registry-ssl-cert-replace.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+:_content-type: PROCEDURE
+[id="mirror-registry-ssl-cert-replace_{context}"]
+= Replacing mirror registry for Red Hat OpenShift SSL/TLS certificates 
+
+In some cases, you might want to update their SSL/TLS certificates for the the _mirror registry for Red Hat OpenShift_. This is useful in the following scenarios:
```
[the remark this answers is not in the document]
Comment (stevsmit) [reply modules/mirror-registry-ssl-cert-replace.adoc:9]: At some point I swapped users/you, and forgot to take out "their." Thanks!

File: modules/mirror-registry-ssl-cert-replace.adoc [hunk 4]
```diff
@@ -0,0 +1,58 @@
+// module included in the following assembly:
+//
+// * installing-mirroring-creating-registry.adoc
+
+:_content-type: PROCEDURE
+[id="mirror-registry-ssl-cert-replace_{context}"]
+= Replacing mirror registry for Red Hat OpenShift SSL/TLS certificates
```
[the remark this answers is not in the document]
Comment (stevsmit) [reply modules/mirror-registry-ssl-cert-replace.adoc:7]: I have; I should probably do it in a new PR.

**Comment by stevsmit** [comment]
Requesting merge review due to updating various files after peer review.

**Comment by jab-rh** [comment]
@stevsmit, looks like this doesn't cherry-pick cleanly to 4.11.

**Pull request merged**